disco/util/hashmap.py

```diff
@@ -11,7 +11,7 @@ def iter(self):
         return iter(self)
 
     def items(self):
-        return six.iteritems(self)
+        return six.iteritems(super(HashMap, self))
```

Comment:
This is probably what you want; could you also throw a super simple compatibility test in?

```python
def items(self):
    return (self.items() if six.PY3 else self.iteritems())
```